edalize/xcelium.py

```diff
@@ -76,55 +71,55 @@ def get_doc(cls, api_ver):
         if api_ver == 0:
             return {'description' : "Xcelium simulator from Cadence Design Systems",
                     'lists' : [
-                        {'name' : 'xmvhdl_options',
+                        {'name' : 'elab_options',
                          'type' : 'String',
-                         'desc' : 'Additional options for compilation with xmvhdl'},
-                        {'name' : 'xmvlog_options',
+                         'desc' : 'Additional run options for xrun -elaboration'},
+                        {'name' : 'xrun_prefix',
                          'type' : 'String',
-                         'desc' : 'Additional options for compilation with xmvlog'},
-                        {'name' : 'xmsim_options',
+                         'desc' : 'prefix for using submission servers like bjob or nc'},
+                        {'name' : 'run_options',
                          'type' : 'String',
-                         'desc' : 'Additional run options for xmsim'},
-                        {'name' : 'xrun_options',
-                         'type' : 'String',
-                         'desc' : 'Additional run options for xrun'},
+                         'desc' : 'Additional run options for xrun -R'},
                         ]}
 
     def _write_build_rtl_f_file(self, tcl_main):
         tcl_build_rtl  = open(os.path.join(self.work_root, "edalize_build_rtl.f"), 'w')
+        tcl_build_c    = open(os.path.join(self.work_root, "edalize_build_c.f"), 'w')
 
         (src_files, incdirs) = self._get_fileset_files()
         vlog_include_dirs = ['+incdir+'+d.replace('\\','/') for d in incdirs]
 
         libs = []
+
+        print("\n DEFINES::", file=sys.stderr)
+        print(self.vlogdefine, file=sys.stderr)
```

Comment:
Use the logger instead of printing to be inline with other backends

Reply:
thank I will do that.